tools/wheels/upload_wheels.sh

```diff
@@ -0,0 +1,62 @@
+# Copied from numpy version
+# https://github.com/numpy/numpy/blob/main/tools/wheels/upload_wheels.sh
+
+set_travis_vars() {
+    # Set env vars
+    echo "TRAVIS_EVENT_TYPE is $TRAVIS_EVENT_TYPE"
+    echo "TRAVIS_TAG is $TRAVIS_TAG"
+    if [[ "$TRAVIS_EVENT_TYPE" == "push" && "$TRAVIS_TAG" == v* ]]; then
+      IS_PUSH="true"
+    else
+      IS_PUSH="false"
+    fi
+    if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+      IS_SCHEDULE_DISPATCH="true"
+    elif [[ "$TRAVIS_EVENT_TYPE" == "api" ]]; then
+      # Manual CI run, so upload
+      IS_SCHEDULE_DISPATCH="true"
+    else
+      IS_SCHEDULE_DISPATCH="false"
+    fi
+}
+set_upload_vars() {
+    echo "IS_PUSH is $IS_PUSH"
+    echo "IS_SCHEDULE_DISPATCH is $IS_SCHEDULE_DISPATCH"
+    if [[ "$IS_PUSH" == "true" ]]; then
+        echo push and tag event
+        export ANACONDA_ORG="multibuild-wheels-staging"
+        export TOKEN="$SCIPY_STAGING_UPLOAD_TOKEN"
+        export ANACONDA_UPLOAD="true"
+    elif [[ "$IS_SCHEDULE_DISPATCH" == "true" ]]; then
+        echo scheduled or dispatched event
+        export ANACONDA_ORG="scipy-wheels-nightly"
+        export TOKEN="$SCIPY_NIGHTLY_UPLOAD_TOKEN"
+        export ANACONDA_UPLOAD="true"
+    else
+        echo non-dispatch event
+        export ANACONDA_UPLOAD="false"
+    fi
+}
+upload_wheels() {
```

Comment:
For `scipy-wheels-nightly` it might be nice to additionally remove old uploads so that the space in the org doesn't fill up (though `scipy` doesn't take up a huge amount of space compared to some others). c.f. https://github.com/matplotlib/matplotlib/pull/23349 where `matplotlib` only keeps the last 5 uploads.

Reply:
That should probably be separate from this PR though. Maybe a cron job. I think @charris cleans up manually now?

Reply:
Sure, could be a cron job but also no need to run on a schedule if you just run when you upload. Both work, though my current thinking on https://github.com/matplotlib/matplotlib/issues/22757#issuecomment-1210713175 is based on running after upload.